README.md

```diff
@@ -55,15 +55,17 @@ Building
 To build you will need:
 
  * A Java compiler compatible with Java 1.7
```

Comment:
s/1.7/1.8/

Reply:
I don't understand what this comment means

Reply:
Replace 1.7 with 1.8, as the text below (correctly) states that java >= 1.8 is required.

Reply:
Actually, you already replaced that in a previous change so my updated version of this PR will not have this line changed.